web/modules/custom/feedback/feedback.permissions.yml

```diff
@@ -1,12 +0,0 @@
-view feedback:
```

Comment:
use more human friendly permissions names.
e.g. "access feedback form" but not "view feedback"

"view feedback" means the ability to view already left feedbacks but not to leave them